Lib/test/test_sqlite3/test_cli.py

```diff
@@ -200,5 +204,36 @@ def test_color(self):
             self.assertIn('\x1b[1;35mOperationalError (SQLITE_ERROR)\x1b[0m: '
                           '\x1b[35mnear "sel": syntax error\x1b[0m', err)
 
+@requires_subprocess()
+class Completer(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # Ensure that the readline module is loaded
+        # If this fails, the test is skipped because SkipTest will be raised
+        readline = import_module("readline")
+        if readline.backend == "editline":
+            raise unittest.SkipTest("libedit readline is not supported")
+
+    def test_keyword_completion(self):
```

Comment:
Ideally, let's test when there is nothing to complete and in which order they are rendered. Make separate tests for that.